apps/mobile/scripts/build-preview.sh

```
@@ -0,0 +1,48 @@
+
+# Login to EAS (if not already logged in)
+echo "🔐 Checking EAS authentication..."
+if ! eas whoami &> /dev/null; then
+    echo "Please log in to EAS:"
+    eas login
+fi
+
+# Load preview environment
+export NODE_ENV=development
+if [ -f .env.preview ]; then
+    export $(cat .env.preview | grep -v '^#' | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)